spec/models/doctor_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Doctor, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I believe is a good idea to delete files if you are not using it, I noticed you have your specs in the requests folder and all are passing, that's great but keeping this empty  file spec the terminal is showing this warning: 🔽  🔽 
![Captura de Pantalla 2022-03-28 a la(s) 9 21 53](https://user-images.githubusercontent.com/88604987/160432227-47016cb0-ec32-4ca5-ac18-a02d1d83c04d.png)

Kindly remove the file to keep your files structure clean! 🤓

Reply:
Done ✔️